app-backend/batch/src/main/scala/com/azavea/rf/batch/landsat8/ImportLandsat8C1.scala

```diff
@@ -278,6 +259,42 @@ case class ImportLandsat8C1(startDate: LocalDate = LocalDate.now(ZoneOffset.UTC)
     }
   }
 
+  def getRowFootprints(row: Map[String, String], srcProj: CRS, targetProj: CRS):
```

Comment:
Moving the footprint logic, you have to keep the `@SuppressWarnings(Array("TraversableHead"))` otherwise this errors when building

Reply:
derp, couldn't figure out why it was complaining here and not before. :man_facepalming: